specification/subscription/resource-manager/Microsoft.Subscription/preview/2020-01-01-preview/subscriptions.json

```diff
@@ -0,0 +1,469 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-10-01-preview",
+    "title": "SubscriptionClient",
+    "description": "Subscription client provides an interface to create and manage Azure subscriptions programmatically."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Subscription/OperationResults/PurchaseSupport_{subscriptionId}:{purchaseId}": {
+      "get": {
+        "operationId": "GetPurchaseSupportStatus_Get",
```

Comment:
>GetPurchaseSupportStatus_Get [](start = 24, length = 28)

should be just PurchaseSupportStatus_Get

Reply:
OperationId GetPurchaseSupportStatus_Get changed to PurchaseSupportStatus_Get.